src/NLU.DevOps.Luis.Shared/LabeledUtteranceExtensions.cs

```diff
@@ -4,22 +4,27 @@
 namespace NLU.DevOps.Luis
 {
     using System;
-    using System.Collections.Generic;
     using System.Linq;
     using Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.Models;
     using Models;
 
     internal static class LabeledUtteranceExtensions
     {
-        public static JSONUtterance ToJSONUtterance(this Models.LabeledUtterance utterance, IReadOnlyDictionary<string, string> prebuiltEntityTypes)
+        public static JSONUtterance ToJSONUtterance(this Models.LabeledUtterance utterance, LuisSettings luisSettings)
         {
             JSONEntity toJSONEntity(Entity entity)
             {
-                var entityType = prebuiltEntityTypes.TryGetValue(entity.EntityType, out var builtinType)
+                var startPos = entity.StartCharIndexInText(utterance.Text);
```

Comment:
>entity [](start = 31, length = 6)

risk of null entity here?

Reply:
This is an internal class, and LUIS doesn't respond with null entities.